uber/decorators.py

```diff
@@ -25,7 +25,7 @@ def with_check(*args, **kwargs):
             except:
                 pass  # we don't care about unrestricted pages for this version
             else:
-                sa.Tracking.track_pageview(cherrypy.request.path_info, cherrypy.request.query_string)
+                sa.PageViewTracking.track_pageview(cherrypy.request.path_info, cherrypy.request.query_string)
```

Comment:
Ha, I wondered about this while I was editing this code. Didn't want to dig too deep into it at the time though.